src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.cs

```diff
@@ -374,9 +385,10 @@ internal static SyntaxTree CreateWithoutClone(CSharpSyntaxNode root)
             string path = "",
             Encoding encoding = null,
             ImmutableDictionary<string, ReportDiagnostic> diagnosticOptions = null,
+            bool? isGeneratedCode = null,
```

Comment:
Please add a backcompat overload

Reply:
Ah, I see you added these below. Sorry for the spam

---
In reply to: [346550550](https://github.com/dotnet/roslyn/pull/39051#discussion_r346550550) [](ancestors = 346550550)